src/deluge/processing/stem_export/stem_export.cpp

```diff
@@ -246,14 +250,23 @@ int32_t StemExport::disarmAllInstrumentsForStemExport() {
 				else {
 					output->exportStem = false;
 				}
-				output->mutedInArrangementModeBeforeStemExport = output->mutedInArrangementMode;
```

Comment:
better to leave this logic alone and skip the disarming step entirely using a new export type